pkg/targets/targetspb/rpc.proto

```diff
@@ -67,10 +67,11 @@ message ActiveTarget {
     ZLabelSet labels = 2 [(gogoproto.jsontag) = "labels", (gogoproto.nullable) = false];
     string scrapePool = 3 [(gogoproto.jsontag) = "scrapePool"];
     string scrapeUrl = 4 [(gogoproto.jsontag) = "scrapeUrl"];
-    string lastError = 5 [(gogoproto.jsontag) = "lastError"];
-    google.protobuf.Timestamp lastScrape = 6 [(gogoproto.jsontag) = "lastScrape", (gogoproto.stdtime) = true, (gogoproto.nullable) = false];
-    double lastScrapeDuration = 7 [(gogoproto.jsontag) = "lastScrapeDuration"];
-    TargetHealth health = 8 [(gogoproto.jsontag) = "health"];
+    string globalUrl = 5 [(gogoproto.jsontag) = "globalUrl"];
```

Comment:
We missed this in the review as this field doesn't exist in Prometheus doc https://prometheus.io/docs/prometheus/latest/querying/api/#targets.
Thanks for catching this!

Reply:
I just caught this because it was required in UI haha. Thanks for the review :)